examples/file_server.md

```diff
@@ -22,42 +22,51 @@ memory.
 **Command:** `deno run --allow-read --allow-net file_server.ts`
 
 ```ts
+import * as Path from "https://deno.land/std@$STD_VERSION/path/mod.ts";
```

Comment:
nit: I think non capital `path` is more common for namespace variable

Reply:
Just updated, thanks for the feedback.